### DISCUSSION TITLE
视频默认720p，能否设置默认画质

### DISCUSSION THREAD

**Discussion by dot-super** [the1812/Bilibili-Evolved#3970] [功能建议 / Ideas]
搜了讨论，发现总是有人断断续续有这个问题，现在我也遇到了，且有人说b站能够记忆画质，无论是关掉所有脚本测试，还是删掉所有cookie重新登录测试，都无法记忆，需要自己重新选择，不知道问题出在哪里

搜了搜其他脚本，也不起作用，只能来这里试试了

**Comment by Geeyun-JY3** [comment]
#3898
